js/actors/grapple.js

```diff
@@ -10,24 +10,27 @@ class Grapple extends Actor {
     constructor(player, targetX, targetY, length) {
         super(player.getCenterX(), player.getCenterY(), 48, 40, "claw");
         this.hitboxes = [new Hitbox(this, Math.round(this.width / 2) - 10, Math.round(this.height / 2) - 10, 20, 20)];
-        this.x -= this.width / 2;
-        this.y -= this.height / 2;
         this.player = player;
         this.state = GrappleState.EXTENDING;
         this.length = length;
         this.extendSpeedFactor = 0.07;
         this.returnAcceleration = 2;
         this.returningSpeed = 0;
+
+        this.xOrigin = Math.round(48 / 2);
```

Comment:
should reference this.width and this.height instead of numbers

Reply:
actually could just call `this.getCenterX()` and `this.getCenterY()`

Reply:
most rotated objects will rotate about their center, so i've added some default values to `Sprite` and removed them from here